def-riscv-insns.tex

```diff
@@ -26,6 +26,11 @@
 \rvcherisrcdest[name=CGetFlags]{7}{rd}{cs1}
 \rvcherisrcdest[name=CGetAddr]{F}{rd}{cs1}
 
+% Mark as noref and reserved until landing of
+% https://github.com/CTSRD-CHERI/sail-cheri-riscv/pull/55
+\rvcherisrcdest[name=CGetHigh,noref,tablesuffix=\rvcherireservedfootnotemark]{1C}{cd}{cs1}
```

Comment:
I don't understand this encoding, it seems to be away from everything in the middle of nowhere (whilst CSetHigh makes sense as the encoding after CRelocate)

Reply:
That's fair; indeed 1C and 1D were going to be their own new island for these accessors.  How about 17 instead?